src/main/webapp/WEB-INF/jsp/submitFeedback.jsp

```diff
@@ -20,91 +20,147 @@
+            <p><form:errors path="feedback"/></p>
+        </div>
+        </c:if>
+    </spring:bind>
+
+    <div data-role="fieldcontain">
+    <fieldset data-role="controlgroup" id="${n}answer">
+        <div class="radio radio-success">
+          <input type="radio" id="yes" name="like" value="YES"/>
+          <label for="yes" class="portlet-form-field-label"><spring:message code="feedback.answer.yes"/></label>
+        </div>
+        <div class="radio radio-danger">
+          <input type="radio" id="no" name="like" value="NO"/>
+          <label for="no" class="portlet-form-field-label"><spring:message code="feedback.answer.no"/></label>
+        </div>
+        <div class="radio radio-warning">
+          <input type="radio" id="maybe" name="like" value="MAYBE"/>
+          <label for="maybe" class="portlet-form-field-label"><spring:message code="feedback.answer.maybe"/></label>
+        </div>
+    </fieldset>
+    </div>
+
+     <p>
+         <label class="portlet-form-field-label" for="${n}feedback"><spring:message code="feedback.form.suggestion"/></label>
+         <textarea id="${n}feedback" name="feedback" class="form-control" rows="${feedbackRows}" style="width:${feedbackWidth}"></textarea>
+
+     </p>
+     <div id="${n}limit" style="margin-bottom: 12px;"></div>
+
+     <p>
+        <fieldset data-role="controlgroup">
+            <div class="checkbox">
+               <input type="checkbox" id="${n}anonymous" name="anonymous" value="true"/>
+               <label for="${n}anonymous" class="portlet-form-field-label"><spring:message code="feedback.form.anonymous"/></label>
+            </div>
+        </fieldset>
+     </p>
+
+     <input id="${n}useragentstring" type="hidden" name="useragent"/>
+     <input id="${n}feedbacktabname" type="hidden" name="tabname"/>
+
+     <p>
+         <input type="submit" id="${n}submitfeedback" class="feedback-submit-button btn btn-default ui-btn-hidden" disabled="disabled" aria-disabled="true" value="<spring:message code="feedback.form.submit"/>">
+     </p>
 
 </form:form>
-
-<script type="text/javascript">
+</div>
+
+<script type="text/javascript"><rs:compressJs>
+   // Enable submitFeedback work here only on Mobile
+    var $ = up.jQuery;
+    $(document).bind("pageinit", function (e) {
+        var $page = $(e.target);
+        $page.find("input:radio[name=like]").change(function () {
+            $(this).checkboxradio("refresh");
+            $page.find("input:radio[name=like]").each(function (index, element) {
+                var $radio = $(this);
+                var $label = $radio.next();
+                if ($label.hasClass("ui-radio-on")) {
+                    //$radio.prop("checked", true);
+                    // DEBUG radio checked
+                    //alert("radio checked");
+                    if ($('#${n}submitfeedback').attr('disabled')) $('#${n}submitfeedback').removeAttr('disabled');
```

Comment:
@ChristianMurphy 
ok tomorrow, I'll :art:
At the time of the PR, the google styles code guidelines weren't used and recommended.
👍